jenkins-scripts/dsl/_configs_/OSRFSourceCreation.groovy

```diff
@@ -140,33 +140,90 @@ class OSRFSourceCreation
     job.with
     {
       publishers {
-        postBuildScripts {
-          steps {
-            conditionalSteps {
-             condition {
-              not {
-                expression('none|None|^$','${ENV,var="UPLOAD_TO_REPO"}')
-                }
-              }
-              steps {
-                // Invoke repository_uploader
-                downstreamParameterized {
-                  trigger(repository_uploader_jobname) {
-                    parameters {
-                      currentBuild()
-                      predefinedProps([PROJECT_NAME_TO_COPY_ARTIFACTS: '${JOB_NAME}',
-                                       PACKAGE_ALIAS: '${PACKAGE_ALIAS}',
-                                       S3_UPLOAD_PATH: "${Globals.s3_releases_dir(package_name)}/"])  // relative path with a final /
-                      propertiesFile(properties_file)  // S3_FILES_TO_UPLOAD
+        postBuildScript {
```

Comment:
I can not find this `postBuildScript` in the Jenkins DSL API in https://jenkinsci.github.io/job-dsl-plugin/# . The local generation can not work. Do you know if it is part of any plugin?

Reply:
IIRC it is: https://plugins.jenkins.io/postbuildscript/